x-pack/plugins/security_solution/public/management/components/endpoint_responder/lib/console_commands_definition.ts

```diff
@@ -136,6 +137,16 @@ const COMMENT_ARG_ABOUT = i18n.translate(
   { defaultMessage: 'A comment to go along with the action' }
 );
 
+const commandCommentArgument = (): { comment: CommandArgDefinition } => {
```

Comment:
I made it a function because I did not want the same object references to be used on all commands - each command should have its own instance of the object so that if one is (for whatever reason) mutated somewhere, only that command will be impacted by it.

I could have made this a `const` and used `Object.freeze()` on it then spread that into each command, but that would have been more cumbersome do the nested object.